lib/mindwendel_web/templates/error/error_page.html.eex

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name='viewport' content='width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0' >
+    <title>mindwendel</title>
+    <link phx-track-static rel="stylesheet" href="<%= Routes.static_path(@conn, "/css/app.css") %>"/>
+    <script defer phx-track-static type="text/javascript" src="<%= Routes.static_path(@conn, "/js/app.js") %>"></script>
+  </head>
+
+  <body class="bg-light">
+    <nav class="navbar sticky-top navbar-expand-lg navbar-light bg-white border-bottom mb-2">
+      <div class="container">
+        <a class="navbar-brand mb-0 h1" href="#">
+          <%= img_tag(Routes.static_path(@conn, "/images/mindwendel_logo_small.png")) %>
+        </a>
+
+        <button
+          class="navbar-toggler"
+          type="button" data-bs-toggle="collapse"
+          data-bs-target="#navbarDropdownToggler"
+          aria-controls="navbarDropdownToggler"
+          aria-expanded="false"
+          aria-label="Toggle navigation" >
+          <span class="navbar-toggler-icon"></span>
+        </button>
+
+        <div class="collapse navbar-collapse" id="navbarDropdownToggler">
+          <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+            <li class="nav-item">
+              <a class="nav-link" aria-current="page" href="/">New brainstorming</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+
+    <main role="main" class="container" id="main-container">
+      <div class="text-center">
+        <h1 style="font-size: 8rem;">
+          <%= @status |> Integer.to_string |> String.at(0) %>
```

Comment:
this only works for error messages with a zero in the middle, or am i mistaken? What if you experience an error like 418 "I am a teapot" ? ;) Better example: 410 for "gone".

Reply:
Yes, I was aware of this. As we only capture `400`, `404`, `500` it would have been ok.

Reply:
Done.